tmt/steps/discover/fmf.py

```diff
@@ -544,8 +544,9 @@ def do_the_discovery(self, path: Optional[Path] = None) -> None:
                     'git', 'log', '--format=', '--stat', '--name-only', f"{modified_ref}..HEAD"
                     ), cwd=self.testdir)
             if output.stdout:
-                directories = [os.path.dirname(name) for name in output.stdout.split('\n')]
-                modified = {f"^/{re.escape(name)}" for name in directories if name}
+                directories = [Path(name).parent for name in output.stdout.split('\n')]
```

Comment:
`.splitlines()`, while we're at it? :)

Reply:
In a separate patch? Because :)

```
$ grep -r "split(['\"]\\\n" tmt
tmt/export/nitrate.py:        for line in cast(str, field.get('environment')).split('\n'):
tmt/steps/discover/fmf.py:                directories = [os.path.dirname(name) for name in output.stdout.split('\n')]
tmt/utils/yaml.py.bak:        for paragraph in value.rstrip().split('\n\n'):
tmt/utils/yaml.py.bak:        yield from value.rstrip().split('\n')
tmt/utils/yaml.py.bak:    for line in result.stdout.split('\n'):
tmt/utils/yaml.py.bak:    for line in result.stdout.split('\n'):
tmt/utils/yaml.py.bak:        for line in content.split("\n"):
tmt/utils/yaml.py.bak:        remotes = output.stdout.split('\n')
tmt/utils/git.py:    for line in result.stdout.split('\n'):
tmt/utils/git.py:    for line in result.stdout.split('\n'):
tmt/utils/git.py:        remotes = output.stdout.split('\n')
tmt/utils/structured_field.py:        for line in content.split("\n"):
tmt/utils/__init__.py:        output_lines = output.strip().split('\n')
tmt/utils/__init__.py:        for paragraph in value.rstrip().split('\n\n'):
tmt/utils/__init__.py:        yield from value.rstrip().split('\n')
tmt/utils/__init__.py:        remotes = output.stdout.split('\n')
tmt/result.py:            iterator = iter(reversed(log.split("\n")))
=
```

Reply:
As you wish, although it's "just" 9 other places, you have some `.bak`s and `\n\n` there. :)